spring-integration-core/src/main/java/org/springframework/integration/transaction/TransactionSynchronizationFactoryBean.java

```diff
@@ -171,6 +174,22 @@ public DefaultTransactionSynchronizationFactory getObject() {
 		if (this.channelResolver == null) {
 			this.channelResolver = new BeanFactoryMessageChannelDestinationResolver(this.beanFactory);
 		}
+<<<<<<< HEAD
```

Comment:
Merging conflict.

Reply:
Not sure what happened there, my bash history says I added it before continuing the rebase.